integration-tests/src/test_loop/tests/state_sync.rs

```diff
@@ -235,18 +240,18 @@ fn send_txs_between_shards(
     }
 }
 
-// Check that no block with height `skip_sync_block_height` made it on the canonical chain, so we're testing
+// Check that no block with height `skip_block_height` made it on the canonical chain, so we're testing
 // what we think we should be.
-fn assert_fork_happened(env: &TestLoopEnv, skip_sync_block_height: BlockHeight) {
+fn assert_fork_happened(env: &TestLoopEnv, skip_block_height: BlockHeight) {
```

Comment:
Is this sufficient to check that fork happened? Just skipping a block isn't the same as a fork.

Reply:
Yea true, added a check that at least one of them knows about a block w height `skip_block_height`